src/main/java/com/internet/shop/controller/order/CreateOrderController.java

```diff
@@ -0,0 +1,38 @@
+package com.internet.shop.controller.order;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.OrderService;
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/orders/create")
+public class CreateOrderController extends HttpServlet {
```

Comment:
```suggestion
public class CompleteOrderController extends HttpServlet {
```
it's better to keep naming consistent

Reply:
done